app/src/main/java/com/myvetpath/myvetpath/Client.java

```diff
@@ -0,0 +1,78 @@
+package com.myvetpath.myvetpath;
+
+import java.sql.Date;
+
+public class Client {
+    //Table Fields
+    public static final String TABLE_NAME = "Client";
+    public static final String COLUMN_ID = "ClientID";
+    public static final String COLUMN_FIRSTNAME = "FirstName";
+    public static final String COLUMN_LASTNAME = "LastName";
+    public static final String COLUMN_PHONENUMBER = "PhoneNumber";
+    public static final String Column_EMAIL = "Email";
+
+    //Create Table String
+    public static final String CREATE_TABLE =
+            "CREATE TABLE " + TABLE_NAME + "(" + COLUMN_ID +
+                    "INTEGER PRIMARY KEY, "
```

Comment:
I think this should be fixed in Brandon's pull but there has to be a space before INTEGER PRIMARY KEY.